src/code-templates/.prettierrc.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   trailingComma: "es5",
   tabWidth: 2,
-  semi: false,
+  semi: true,
```

Comment:
@DanielGluskin @rluvaton The main idea of Prettier is "we set good default so you don't deal with it"

Should we really express an opinion here?